### ISSUE TITLE
Alright good... We're away...

### ISSUE THREAD

**Issue by procrasti** [orionblastar/FreeK666backup#2]
At least I can push issues at you.

Somewhere along the line, you've lost that this is a fork of procrasti/k666... I don't really like that tbh...

I know it's been annoying... but could you recreate orionblaster/k666... That should be your freek666.

The main annoyance to me is that I can't easily merge it back into k666 and keep track of changes.

Are these all the changes you had in your old k666 repository?

**Comment by procrasti** [comment]
So, orionblaster/k666 should be a fork of procrasti/k666... then FreeK666 should really be a fork of orionblaster/k666.

**Comment by orionblastar** [comment]
It is confusing, I had K666 that had files in it that you didn't want. I don't know how to remove them from git history so I deleted the repository and found a better git ignore and then forked your K666 into it and started over from scratch. I made a backup of the files at FreeK666 which is now FreeK666backup because when I get a stable release I will copy all files there and then push changes.

Thing is I should have made a FreeK666 branch in K666 and had a better git ignore file. I made a lot of new programmer mistakes and didn't know how branches are used or made.
